extensions/measurement-tracking/src/viewports/TrackedCornerstoneViewport.js

```diff
@@ -408,6 +419,7 @@ function _getCornerstoneStack(displaySet, dataSource) {
   return stack;
 }
 
+// TODO -> disabled double click for now: onDoubleClick={_onDoubleClick}
```

Comment:
True. Some functionality should probably be disabled if _isPlaying_

Reply:
This TODO was added previously, any impact? I just moved it a couple of lines